rest-api/rest-api.php

```diff
@@ -0,0 +1,637 @@
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_belief';
+                            $int_count++;
+
+                            break;
+
+                        case 'can share':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_can_share';
+                            $int_count++;
+
+                            break;
+
+                        case 'baptized':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_baptized';
+                            $int_count++;
+
+                            break;
+
+                        case 'has bible':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_has_bible';
+                            $int_count++;
+
+                            break;
+
+                        case 'studying':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_reading_bible';
+                            $int_count++;
+
+                            break;
+
+                        case 'profession':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_belief';
+                            $int_count++;
+
+                            break;
+
+                        case 'can share':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_can_share';
+                            $int_count++;
+
+                            break;
+
+                        case 'sharing':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_sharing';
+                            $int_count++;
+
+                            break;
+
+                        case 'baptizing':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_baptizing';
+                            $int_count++;
+
+                            break;
+
+                        case 'in group':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_in_group';
+                            $int_count++;
+
+                            break;
+
+                        case 'starting groups':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_planting';
+                            $int_count++;
+
+                            break;
+
+                        case 'baptized':
+
+                            $fields_map['milestones']['values'][$int_count]['value'] = 'milestone_baptized';
+                            $int_count++;
+
+                            break;
+
+
+                    }
+                }
+
+            }
+        }
+        else //No milestones
+            {
+            $fields_map['milestones']['values'][1]['value'] = 0;
```

Comment:
What is the intention with this code? I noticed that if I didn't send the `milestones` property in the API request, the activity comment in DT said "0 added to Faith Milestones".